doc/source/serve/architecture.md

```diff
@@ -49,7 +53,7 @@ Each replica maintains a queue of requests and executes requests one at a time,
 using `asyncio` to process them concurrently. If the handler (the deployment function or the `__call__` method of the deployment class) is declared with `async def`, the replica will not wait for the
```

Comment:
Hmm, this sounds right. It's basically saying `async def` will be running in asynchronous fashion and not blocking each call. But can talk offline to see if it can be improved.